src/models/WeekDateObject.ts

```diff
@@ -23,22 +23,20 @@ export default class WeekDateObject implements AtomicDateObject {
   private dateTypeFormat: DateTypeFormat
 
   static createInnerHTML = (week: AtomicDateObject[], split = false, month?: number): string => {
-    return week.reduce((accumulator: string, ado: AtomicDateObject, index: number) => {
+    let innerHTML = ''
+
+    week.forEach((ado: AtomicDateObject, index: number) => {
       let newElement = `<div class="weekday" data-ado-index=${index}>${ado.viewString}</div>`;
-      /**
-       * When the week is split (some days belong to one month and the rest belong to
-       * another month) only some of the days are filled.
-       * The ones that don't belong to the current month are empty.
```

Comment:
This comment seemed helpful. Maybe the variable should be `splitWeek`?

Reply:
good call!